src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx

```diff
@@ -223,6 +294,31 @@ function WorkspaceMemberDetailsPage({personalDetails, policy, route}: WorkspaceM
                             onRoleChange={changeRole}
                             onClose={() => setIsRoleSelectionModalVisible(false)}
                         />
+                        {policy?.areExpensifyCardsEnabled && (
+                            <>
+                                <View style={[styles.ph5]}>
+                                    <Text style={StyleUtils.combineStyles([styles.sidebarLinkText, styles.optionAlternateText, styles.textLabelSupporting])}>
```

Comment:
I think we don't need `StyleUtils.combineStyles` here, could you remove and test it?

Reply:
It breaks the font size